frappe/core/doctype/system_settings/system_settings.json

```diff
@@ -507,6 +507,170 @@
    "translatable": 0,
    "unique": 0
   },
+  {
```

Comment:
Why does the JSON look like the old version of frappe, Are you sure you were using the latest version for development?

For example https://github.com/frappe/frappe/blob/f9cd91cca21f419242dc81e5e82811752921bdac/frappe/website/doctype/website_theme/website_theme.json

Reply:
thanks for pointing this out, we're going to use the latest version.